tests/network/vmi_subdomain.go

```diff
@@ -88,10 +95,10 @@ var _ = SIGDescribe("Subdomain", func() {
 
 			Expect(assertFQDNinGuest(vmi, expectedFQDN)).To(Succeed(), "failed to get expected FQDN")
 		},
-			Entry("with Masquerade binding and subdomain", fedoraMasqueradeVMI, subdomain),
-			Entry("with Bridge binding and subdomain", fedoraBridgeBindingVMI, subdomain),
-			Entry("with Masquerade binding without subdomain", fedoraMasqueradeVMI, ""),
-			Entry("with Bridge binding without subdomain", fedoraBridgeBindingVMI, ""),
+			Entry("with Masquerade binding and subdomain and hostname", fedoraMasqueradeVMI, subdomain, hostname),
```

Comment:
but not we dont test subdomain without host name at all, i think at least one of them should do it ?

Reply:
True, this is the impact of modifying existing tests compare to just adding tests as I did in the first patch.
But I do agree we need coverage for subdomain without hostname.